ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -1048,6 +1048,24 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= [Host_selectors]
       }
     )
+  ; ( "host-start-ssh"
```

Comment:
Would `host-ssh-start/stop` be the better name? This follows the hierarchical structure we use.